src/components/input/LocationAddress.vue

```diff
@@ -0,0 +1,200 @@
+<template>
+  <label
+    :class="{ focused: isInputFocused }"
+    v-overlay-container="{ togglePropName: 'displaySuggestions' }"
+    v-navigable-container="{ cursorPropName: 'cursorIndex' }"
+  >
+    <input
+      type="text"
+      @input="onInputInput"
+      @focus="onInputFocus"
+      @blur="onInputBlur"
+      @keydown="onInputKeyDown"
+      v-model="query"
+      :placeholder="placeholder"
+    />
+    <ul v-if="displaySuggestions">
+      <li
+        v-for="(suggestion, index) in suggestions"
+        :key="suggestion.id"
+        @click="onSuggestionClick(index)"
+        :class="{ cursor: index === cursorIndex }"
+      >
+        <span>{{ suggestion.address }}</span>
+      </li>
+    </ul>
+  </label>
+</template>
+<style scoped lang="scss">
+@import "../../style/variables";
+
+label {
+  display: flex;
+  flex-direction: row;
+  position: relative;
+  color: $greyscale-1;
+
+  &::after {
+    content: " ";
+    display: block;
+    position: absolute;
+    top: 0;
+    right: 0;
+    width: 25%;
+    height: calc(100% - 2px);
+    background: linear-gradient(to right, rgba(255, 255, 255, 0), white);
+  }
+
+  &.focused::after {
+    content: none;
+  }
+}
+
+input {
+  font-size: 12px;
```

Comment:
Won't this font size have iOS zoom the page in when the field gets focus?

Reply:
It will, yes. This issue aside, the font-size really is too small, and this is coming from me, a guy with no sight problems. These sizes are taken from the design exports and while they look fine there, they really don't work for devices with a higher DPI. We just received the fonts from the client last week, so I will re-work typography from the ground up. You can look at this value as being a placeholder.